staging/src/k8s.io/api/core/v1/well_known_labels.go

```diff
@@ -19,10 +19,10 @@ package v1
 const (
 	LabelHostname = "kubernetes.io/hostname"
 
-	LabelZoneFailureDomain       = "failure-domain.beta.kubernetes.io/zone"
-	LabelZoneRegion              = "failure-domain.beta.kubernetes.io/region"
-	LabelZoneFailureDomainStable = "topology.kubernetes.io/zone"
-	LabelZoneRegionStable        = "topology.kubernetes.io/region"
+	LabelFailureDomainBetaZone   = "failure-domain.beta.kubernetes.io/zone"   // deprecated, remove after 1.20
```

Comment:
Is this breaking Go compatibility? I think that's why we kept the beta label pointing to `LabelZoneFailureDomain` and named the new topology labels `LabelZoneFailureDomainStable`.

Reply:
It's not clear to me if Go-level compat is a goal or not?

Reply:
Will retain old names